.eslintrc.json

```diff
@@ -1,5 +1,9 @@
 {
 
-   "extends": "eslint-config-silvermine/node"
+   "extends": "eslint-config-silvermine/node",
+
+   "rules": {
+      "no-console": 0
```

Comment:
Funny, I had initially put `false`, and then got this error:

```
Configuration for rule "no-console" is invalid:
Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed 'false').
```

So then I put zero and forgot to look what we used elsewhere. Apparently they're equivalent. Changed anyway to be consistent.